core-bundle/src/Security/Voter/DataContainer/DefaultDcaVoter.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Security\Voter\DataContainer;
+
+use Contao\DataContainer;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\Voter;
+
+/**
+ * By default, the Contao back end is fully accessible unless a developer wants to have specific
+ * permissions. That's why this voter is implemented with a very low priority, so it allows everything
+ * in the back end as the last voter in case no other voter decided to deny access before.
+ */
+class DefaultDcaVoter extends Voter
```

Comment:
I would probably not implement the `Voter` class to simplify the implementation.

Reply:
Then I have to implement the foreach loop to vote on all `$attributes` myself. Why would I?

Reply:
it would looks simpler to me 🙃 

```
class DefaultDcaVoter implements VoterInterface
{
    public function vote(TokenInterface $token, $subject, array $attributes)
    {
        foreach ($attributes as $attribute) {
            if (str_starts_with($attribute, DataContainer::PERMISSION_PREFIX)) {
                return self::ACCESS_GRANTED
            }
        }

        return self::ACCESS_ABSTAIN
    }
}
```

Reply:
Done (but better ;)) in dfb2f3b8d6c0fc1bbad763e30ce300fe08659ed8.